pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -325,6 +324,10 @@ public MetadataStoreExtended createConfigurationMetadataStore() throws MetadataS
                         .build());
     }
 
+    public void closeMetadataServiceSession() throws IOException {
```

Comment:
this is a public method, is it worth to add a javadoc ?

Reply:
👍